letsencrypt/plugins/manual.py

```diff
@@ -26,13 +26,17 @@
 class Authenticator(common.Plugin):
     """Manual Authenticator.
 
-    .. todo:: Support for `~.challenges.DVSNI`.
-
+    This plugin requires user's manual intervention in setting up a HTTP
+    server for solving SimpleHTTP challenges and thus does not need to be
+    run as a privilidged process. Alternatively shows instructions on how
+    to use Python's built-in HTTP server and, in case of HTTPS, openssl
+    binary for temporary key/certificate generation.
     """
     zope.interface.implements(interfaces.IAuthenticator)
     zope.interface.classProvides(interfaces.IPluginFactory)
+#    hidden = True
```

Comment:
I understand this was done so you could test the UI, but make sure this line isn't commented before we merge.